discord/user.py

```diff
@@ -377,6 +381,39 @@ def display_banner(self) -> Optional[Asset]:
         """
         return self.banner
 
+    @property
+    def banner_colour(self) -> Optional[Colour]:
+        """Optional[:class:`Colour`]: Returns the user's banner colour, if applicable.
+
+        There is an alias for this named :attr:`banner_color`.
+
+        .. versionadded:: 2.0
+
+        .. note::
+
+            This information is only available via :meth:`Client.fetch_user`.
+        """
+        if self._banner_colour is None:
+            return None
+        return Colour(self._banner_colour)
+
+    @property
+    def banner_color(self) -> Optional[Colour]:
```

Comment:
Why is the code doubled?

Reply:
I did the same as the library did with `accent_color`. There is 2 properties, same value, different name. Color is the spelling used in the United States. Colour is used in other English-speaking countries.